package.json

```diff
@@ -40,7 +40,7 @@
     "test:widgets": "yarn workspace @webex/cc-station-login test:unit && yarn workspace @webex/cc-user-state test:unit",
     "build": "yarn run build:store && yarn run build:widgets",
     "build:store": "yarn workspace @webex/cc-store build:src",
-    "build:widgets": "yarn workspace @webex/cc-station-login build:src && yarn workspace @webex/cc-user-state build:src",
+    "build:widgets": "yarn workspace @webex/cc-station-login build:src && yarn workspace @webex/cc-user-state build:src && yarn workspace @webex/cc-widgets build:src ",
```

Comment:
i think you can use the workspace foreach command rather then writing all individually, you can run them in parallel as well

Reply:
Updated the command. We can build station-login and user-state in parallel. But since cc-widgets depends on the later packages it needs to be build after them.

Reply:
We can run them in paralle. used the --topological flag which ensures that dependencies are build in a parallel first and then the cc-widgets.